### ISSUE TITLE
[MS] Re-export all stuff define in `plugins/libparsec/definition.d.ts` in `plugins/libparsec/index.ts`

### ISSUE THREAD

**Issue by FirelightFlagboy** [Scille/parsec-cloud#5168]
From @touilleMan [comment](https://github.com/Scille/parsec-cloud/pull/5141#pullrequestreview-1617718313)

**Comment by touilleMan** [comment]
I don't get the difference with #5167 :thinking:

**Comment by FirelightFlagboy** [comment]
I provide the following example:

#5167 is:

```rust
mod foo {
  pub type A = ();
  pub type B = ();
}
```

#5168 (the current one) is:

```rust
pub use foo::*;
```

**Comment by touilleMan** [comment]
What we want is to expose `libparsec` as a flat module, so `libparsec/definitions` is only an implementation detail (and hence should have never been used in the gui code in the first place)

So I guess we can close #5167

**Comment by FirelightFlagboy** [comment]
#5167 is fixed by #5141, so it will be closed once merged

**Comment by FirelightFlagboy** [comment]
Useful comment: https://github.com/Scille/parsec-cloud/issues/5167#issuecomment-1715861592
